core/arch/arm/kernel/boot.c

```diff
@@ -1061,6 +1061,17 @@ void __weak boot_init_primary_late(unsigned long fdt __unused,
 	} else {
 		init_tee_runtime();
 	}
+}
+
+/*
+ * Note: this function is weak just to make it possible to exclude it from
+ * the unpaged area.
+ */
+#ifdef ARM64
+DISABLE_PAUTH
```

Comment:
Could this macro be renamed `__pauth` and defined also in _arm32.h_ as `#define __pauth`? It would prevent need for `#ifdef ARM64`/`#endif`, e.g. here:
```c
void __weak __pauth boot_init_primary_finalcalls(void)

Reply:
Fixed